plugins/plugin-chart-echarts/src/utils/series.ts

```diff
@@ -86,3 +88,63 @@ export function extractGroupbyLabel({
     .map(val => formatSeriesName(datum[val], { numberFormatter, timeFormatter }))
     .join(', ');
 }
+
+export function getLegendProps(
+  type: LegendType,
+  orientation: LegendOrientation,
+  show: boolean,
+): echarts.EChartOption.Legend {
+  const legend: echarts.EChartOption.Legend = {
+    orient:
+      orientation === LegendOrientation.Top || orientation === LegendOrientation.Bottom
```

Comment:
I would prefer using includes

```suggestion
      [LegendOrientation.Top, LegendOrientation.Bottom].includes(orientation)
```